### ISSUE TITLE
pandas.test() FAILS after fresh anaconda install

### ISSUE THREAD

**Issue by randomgambit** [pydata/pandas#12795]
Hello,

Please see here http://stackoverflow.com/questions/36409183/pandas-test-fails-after-fresh-anaconda-install?noredirect=1#comment60435297_36409183.

I have downloaded a fresh anaconda installer, and ran pandas.test() afterwards. My computer cannot connect to the internet but I am getting failures for tests that do not seem to need connectivity. Is this worrisome or my installation is OK?

Many thanks!

**Comment by jreback** [comment]
well, pls post ``pd.show_versions()``

**Comment by randomgambit** [comment]
sure. thanks for helping

```
pd.show_versions()

INSTALLED VERSIONS
------------------
commit: None
python: 2.7.11.final.0
python-bits: 64
OS: Windows
OS-release: 7
machine: AMD64
processor: Intel64 Family 6 Model 63 Stepping 2, GenuineIntel
byteorder: little
LC_ALL: None
LANG: en_US

pandas: 0.18.0
nose: 1.3.7
pip: 8.1.1
setuptools: 20.3
Cython: 0.23.4
numpy: 1.10.4
scipy: 0.17.0
statsmodels: None
xarray: None
IPython: 4.1.2
sphinx: 1.3.5
patsy: 0.4.0
dateutil: 2.5.1
pytz: 2016.2
blosc: None
bottleneck: 1.0.0
tables: 3.2.2
numexpr: 2.5
matplotlib: 1.5.1
openpyxl: 2.3.2
xlrd: 0.9.4
xlwt: 1.0.0
xlsxwriter: 0.8.4
lxml: 3.6.0
bs4: 4.4.1
html5lib: None
httplib2: None
apiclient: None
sqlalchemy: 1.0.12
pymysql: None
psycopg2: None
jinja2: 2.8
boto: 2.39.0
```

**Comment by jreback** [comment]
a couple of tests fail on some platforms, mainly because of some newer deps that anaconda picked up. These are fixed in upcoming 0.18.1 (a few weeks yet for release). But are really tiny things and no big deal.

**Comment by randomgambit** [comment]
ok thank you jreback!
